CONTRIBUTING.md

```diff
@@ -42,6 +42,28 @@ You can then fix the issue or implement the feature on your side and contribute
 Of course, if you are already using F3D and want to improve it for your specific needs, because you want a feature or found a bug,
 that is definitely possible. Feel free to reach out for guidance by opening an issue or asking on [Discord](https://discord.f3d.app).
 
+## Bounties
+
+The [F3D-APP Foundation](https://annuaire-entreprises.data.gouv.fr/entreprise/f3d-app-foundation-941551624), as a non-profit institution, would like to redistribute some donations to the contributors, for specific issues identified by maintainers as important for the project.
+Such issues have been identified with the ["bounty"]() label. The amount of money for these issues is fairly small (up to a few hundred euros).
+
+In order to be elligible for the bounty you need to:
+ - Be a [returning contributor](doc/dev/GOVERNANCE.md#12-returning-contributors)
+ - Be able to provide an IBAN from a UE country where the money will be donated to
+
+If you are not elligible or are not interested by the donation, this is perfectly fine, you can still work on the issue, but there will be no donation.
```

Comment:
Maybe specify the donation can be donated to a charity if the contributor isn't from UE?